### ISSUE TITLE
MQTTWebsocketTransport missing in framework

### ISSUE THREAD

**Issue by HarrysLapTimer** [novastone-media/MQTT-Client-Framework#495]
It seems that MQTTWebsocketTransport.h is not public for the MQTTClient framework generated. Simply adding the header file (assuming the lib includes the class implementation) to a project doesn't work because the header includes other non-public headers.

- Harald

**Comment by fdstevex** [comment]
The MQTTWebsocketTransport.m file is not included in any of the targets.

**Comment by HarrysLapTimer** [comment]
Yes, thanks. I found this to be the root cause it is not included in the framework. Using CocoaPads actually fixed the problem. MQTTWebsocketTransport is mentioned in the top level documentation prominently. It would probably make sense to add this finding as a comment for other users.
